lib/skills/src/flipCoin.js

```diff
@@ -42,27 +42,43 @@ const skill = ({ controller }) => {
     });
 
     const reply = [results.join(' ')];
+    const timestamp = Math.floor(Date.now() / 1000);
 
-    if (flips > 10) {
-      details.sequences.sort((a, b) => {
-        if (b.length > a.length) return 1;
-        if (b.length < a.length) return -1;
-        return 0;
-      });
+    const sequence = details.sequences.sort((a, b) => {
```

Comment:
We should move this down to where the `if (flips > 10) {` is because it's not needed outside that case. (and it'll just be `undefined` anyways).